components/dashboard/src/hooks/use-analytics-tracking.ts

```diff
@@ -50,4 +58,33 @@ export const useAnalyticsTracking = () => {
         window.addEventListener("click", handleButtonOrAnchorTracking, true);
         return () => window.removeEventListener("click", handleButtonOrAnchorTracking, true);
     }, []);
+
+    useEffect(() => {
+        if (loading || !user || !user.additionalData?.profile?.onboardedTimestamp || !isOrbitalLoaded) {
+            return;
+        }
+
+        console.debug("IDing user");
+        orbital("identify", user.id);
+
+        console.log(`Changing theme to ${isDark ? "dark" : "light"}`);
+        orbital("customConfig", {
```

Comment:
Since this is `window["orbital"]` we should guard against it not existing to avoid runtime errors if there's a problem loading their script.

Reply:
I believe this should be handled for us -> we always create the function independent of loading status here: 
https://github.com/gitpod-io/gitpod/blob/936d7aa39d64d926a563d244b23f0f6376d817b2/components/dashboard/src/hooks/use-orbital.ts#L30

Reply:
@selfcontained what I ended up doing is before completing the orbital loading, we first check if it really exists. If it doesn't, we just do not complete the loading hence orbital never initializes. Thank you for catching this 🙏